incubator/redis-cache/templates/ss.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: "{{ template "fullname" . }}"
+  labels:
+     app: {{ template "fullname" . }}
+     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+     release: "{{ .Release.Name }}"
+     heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: "{{ template "fullname" . }}"
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+        pod.beta.kubernetes.io/init-containers: '[
+           {
+             "name":"sentinel-micro",
+             "image":"{{ .Values.microSentinel.image }}:{{ .Values.microSentinel.tag }}",
+             "imagePullPolicy": "{{ .Values.microSentinel.pullPolicy }}",
+             "args" : ["-service","{{ template "fullname" . }}"],
+             "resources":
+{{ toJson .Values.microSentinel.resources | indent 17 }},
+             "env": [
+               {
+                  "name": "POD_NAMESPACE",
+                  "valueFrom": {
+                     "fieldRef" : {
+                        "apiVersion" : "v1",
+                        "fieldPath" : "metadata.namespace"
+                     }
+                  }
+               }
+             ],
+            "volumeMounts": [
+               {
+                  "name": "config",
+                  "mountPath": "/config"
+               }
+            ]
+           }
+        ]'
+    spec:
+      terminationGracePeriodSeconds: 5
+      affinity:
```

Comment:
Sure agreed.  Same thing here, we could have spotted this few weeks ago, this portion is been there since the first commit.  I would have loved to address this long back and my chart would have gotten merged by now?